osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/DefaultSpinnerDisc.cs

```diff
@@ -124,6 +126,9 @@ protected override void Update()
 
         private void updateStateTransforms(DrawableHitObject drawableHitObject, ArmedState state)
         {
+            if (!(drawableHitObject is DrawableSpinner))
```

Comment:
how can this happen?

Reply:
Because the event is propagated upwards from nested hitobjects.

Reply:
so it fires multiple times on the spinner itself? o_O

Reply:
Yeah:
![image](https://user-images.githubusercontent.com/1329837/94806212-8dd9db80-0428-11eb-977c-3280d1b4388c.png)

Reply:
wow... we may want to check other usages of that.

are you able to change the xmldoc while we're here to make it more obvious that this is the case?

Reply:
Done. Hopefully that's better?